R/fct_SQLite.R

```diff
@@ -163,6 +214,27 @@ db_update <- function(
   cat("Finished updating review data\n")
 }
 
+db_upsert <- function(con, data, common_cols) {
```

Comment:
Even for internal functions, it would be nice to have a roxygen description, which I think helps people who are less familiar with the code. What does it do, what are the inputs, the function arguments, and the expected output? I had to google what 'upsert' means (which probably shows he depth of my SQL knowledge ;) ).